docusaurus/docs/introduction/plugin-types-usage.md

```diff
@@ -0,0 +1,108 @@
+
+### Usage of data-source plugins
+
+Create and configure a Grafana data source when you want to work with data provided by a third-party service for use in Grafana Dashboards, Explore, Alerting, and so on.
+
+Given any installed plugin of type `datasource`, you can create and configure any number of data sources per [Grafana organization](https://grafana.com/docs/grafana/latest/administration/organization-management/#about-organizations). After you create zero to infinity data sources, they are persisted in Grafana's database. 
+
+:::note
+
+To distinguish a Grafana data source from a data-source plugin, we sometimes refer to the latter as a _data-source instance_, that is, a configured Grafana data source with a `plugin id` type of `datasource`.
+
+:::
+
+### Configuration of data sources
+
+Use the [Grafana configuration file](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#configuration-file-location) to configure your data source [`plugin_id`](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#pluginplugin_id).
+
+### Provisioning of data sources
+
+Data sources can also be provisioned using Grafana’s [provisioning features](https://grafana.com/docs/grafana/latest/administration/provisioning/#data-sources) so that you can define Grafana as-code as part of a GitOps approach when Grafana starts up or [on demand](https://grafana.com/docs/grafana/latest/developers/http_api/admin/#reload-provisioning-configurations). 
+
+Note that a plugin of type `datasource` must be installed before it can be provisioned.
+
+### Bundling of dashboards
+
+Data-source plugins can include dashboards by referencing dashboard JSON files (including `property` and `type=dashboard`) within the `plugin.json` file. Grafana puts a dashboard in the `General` folder when it is imported.
+
+## App plugins
+
+Development of app plugins, also known as apps or applications, allows you to create out-of-the-box solutions such as in the [Redis](https://grafana.com/grafana/plugins/redis-app/) app. You can optionally bundle data sources and panels, as well as provide custom pages, [Scenes](https://grafana.com/developers/scenes), and [UI extensions](../ui-extensions/).
+
+### Usage of apps
+
+Configure a Grafana app when you want to leverage or create a tailored monitoring view for a third-party service and optionally use custom pages or UI extensions. Given any installed plugin of type `app`, you can enable them once per [Grafana organization](https://grafana.com/docs/grafana/latest/administration/organization-management/#about-organizations) and they are persisted in Grafana’s database. 
+
```

Comment:
Maybe since we're mentioning this for datasources?

```suggestion

:::note

To distinguish a Grafana app from an app plugin, we sometimes refer to the latter as an _app instance_, that is, a configured Grafana app with a `plugin id` type of `app`.

:::
```

Reply:
done.